dialogue-core/src/main/java/com/palantir/dialogue/core/QueuedChannel.java

```diff
@@ -116,7 +118,8 @@
      */
     @Override
     public ListenableFuture<Response> execute(Endpoint endpoint, Request request) {
-        DeferredCall components = ImmutableDeferredCall.of(endpoint, request, SettableFuture.create());
+        DetachedSpan span = DetachedSpan.start("queued");
+        DeferredCall components = DeferredCall.of(endpoint, request, span, SettableFuture.create());
 
         if (!queuedCalls.offer(components)) {
             return Futures.immediateFuture(RateLimitedResponse.INSTANCE);
```

Comment:
Should we complete the DetachedSpan here? Or are we happy with not recording it at all in this case?

Reply:
Suppose queued doesn't make sense unless we actually do queue something. Disregard!